tester/src/test/java/com/silanis/esl/sdk/examples/CreateAndUpdateSubAccountExampleTest.java

```diff
@@ -0,0 +1,43 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.api.model.AccessibleAccountResponse;
+import com.silanis.esl.api.model.Account;
+import com.silanis.esl.sdk.internal.EslServerException;
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.List;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+import static com.silanis.esl.sdk.examples.CreateAndUpdateSubAccountExample.NAME;
+
+public class CreateAndUpdateSubAccountExampleTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void verifyResult() {
+        CreateAndUpdateSubAccountExample example = new CreateAndUpdateSubAccountExample();
+        List<Account> subAccountList;
+        List<AccessibleAccountResponse> accessibleAccountList;
+
+        try {
+            example.run();
+        } catch (EslServerException e) {
+            thrown.expect(EslServerException.class);
+            thrown.expectMessage(containsString("error.notFound.accountNotFound"));
+            throw e;
+        } finally {
+            subAccountList = example.subAccounts;
```

Comment:
It's quite rare to use finally in unit tests. Usually the test is failed if an exception is thrown unless testing for the exception.

Reply:
the test uses try cache statement to cover "dummy account ID", "non-subaccount enabled account ID" and "full featured account ID" and ensure expected exceptions (NoPermissionException and NoFoundAccountException) and result, against other invalid exceptions.

Reply:
Possible to comment that in the code ?

Reply:
Yes, It's commented.